src/Vcs/Git/GitVcs.cs

```diff
@@ -17,7 +17,7 @@ public void Commit(string csProjFilePath, string message)
                 throw new OperationCanceledException($"Unable to add cs proj file {csProjFilePath} to git index");
             }
 
-            if(!LaunchGitWithArgs($"commit -m \"{message}\""))
```

Comment:
**suggestion**
Perhaps it would be wiser to add a cli-flag to dotnet version cli which allows skipping hooks?